tools/cldr-code/src/main/java/org/unicode/cldr/util/VettingViewer.java

```diff
@@ -53,6 +53,13 @@
 
     private static final boolean DEBUG = false;
 
+    /**
+     * DASHBOARD_USES_BASELINE: the "baseline" info is not used for the Dashboard;
+     * it is possibly still used for Priority Items Summary.
+     * TODO: remove code for baseline where it isn't needed. Reference: https://unicode-org.atlassian.net/browse/CLDR-14877
+     */
+    public static final boolean DASHBOARD_USES_BASELINE = false;
```

Comment:
I've added this constant so that we don't waste time getting baseline value for Dashboard when we don't need it, but we can easily restore the code for getting baseline value if we decide to make Dashboard use baseline again. However, unless we actually have plans for Dashboard to use baseline, I suggest that we simplify the code by removing this constant and simplify the code that depends on it.

Reply:
Never mind -- even though the Dashboard front end doesn't use baseline (a.k.a. "old"), the back end does need it while generating Dashboard data